router/src/main/java/io/micronaut/web/router/exceptions/UnsatisfiedRouteException.java

```diff
@@ -43,6 +47,21 @@ public UnsatisfiedRouteException(Argument<?> argument) {
     }
 
     private static String buildMessage(Argument<?> argument) {
+        Optional<String> bindableOpt = argument.getAnnotationMetadata().getAnnotationNameByStereotype(Bindable.class);
```

Comment:
Why not just use `getAnnotationTypeByStereotype`?

Reply:
Done